nexus/mgs-updates/src/rot_updater.rs

```diff
@@ -192,3 +197,29 @@ impl SpComponentUpdater for RotUpdater {
         &self.log
     }
 }
+
+pub struct ReconfiguratorRotUpdater;
+impl SpComponentUpdateHelper for ReconfiguratorRotUpdater {
+    /// Checks if the component is already updated or ready for update
+    fn precheck<'a>(
+        &'a self,
+        _log: &'a slog::Logger,
+        _mgs_clients: &'a mut MgsClients,
+        _update: &'a PendingMgsUpdate,
+    ) -> BoxFuture<'a, Result<PrecheckStatus, PrecheckError>> {
+        // TODO-K: fill in the precheck
+        todo!()
+    }
+
+    /// Attempts once to perform any post-update actions (e.g., reset the
+    /// device)
+    fn post_update<'a>(
+        &'a self,
+        _log: &'a slog::Logger,
+        _mgs_clients: &'a mut MgsClients,
+        _update: &'a PendingMgsUpdate,
+    ) -> BoxFuture<'a, Result<(), GatewayClientError>> {
+        // TODO-K: fill in the post_update
+        todo!()
+    }
```

Comment:
These will be completed in a follow up PR (step two of https://github.com/oxidecomputer/omicron/issues/7989)